kafkaadmin/topics.go

```diff
@@ -45,3 +46,82 @@ func (c Client) DeleteTopic(ctx context.Context, name string) error {
 	_, err := c.c.DeleteTopics(ctx, []string{name})
 	return err
 }
+
+//GetTopics takes a []*regexp.Regexp and returns a []string of all topic
+//names that match any of the provided regex, and query kafka directrly rather than zk
+func (c Client) GetTopics(ts []*regexp.Regexp) ([]string, error) {
+	ret, er := c.c.GetMetadata(nil, true, FetchMetadaTimeoutMs)
```

Comment:
```suggestion
	metadata, err := c.c.GetMetadata(nil, true, FetchMetadataTimeoutMs)
```
`metadata` o `md` se entiende mas que ret

Reply:
Ok, modificado.